### PR TITLE
Also test on python 3.7

### PR DESCRIPTION
**Pull request by rogiervandergeer**
See #82

### CODE REVIEW

**Comment by rogiervandergeer** [comment]
It appears 3.7 isn't available.

**Comment by koaning** [comment]
strange. it is suggested [here](https://github.com/travis-ci/travis-ci/issues/9815) that this should be a fixed issue. will try it.

**Comment by koaning** [comment]
mhm. strange.

**Comment by koaning** [comment]
Interesting, it seems that python3.7 has different demands for the `setup.py`.

```
Traceback (most recent call last):
  File "setup.py", line 34, in <module>
    'Topic :: Scientific/Engineering :: Artificial Intelligence']
  File "/home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/opt/python/3.7-dev/lib/python3.7/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/opt/python/3.7-dev/lib/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/opt/python/3.7-dev/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages/setuptools/command/install.py", line 67, in run
    self.do_egg_install()
  File "/home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages/setuptools/command/install.py", line 109, in do_egg_install
    self.run_command('bdist_egg')
  File "/opt/python/3.7-dev/lib/python3.7/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/opt/python/3.7-dev/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages/setuptools/command/bdist_egg.py", line 218, in run
    os.path.join(archive_root, 'EGG-INFO'), self.zip_safe()
  File "/home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages/setuptools/command/bdist_egg.py", line 269, in zip_safe
    return analyze_egg(self.bdist_dir, self.stubs)
  File "/home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages/setuptools/command/bdist_egg.py", line 379, in analyze_egg
    safe = scan_module(egg_dir, base, name, stubs) and safe
  File "/home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages/setuptools/command/bdist_egg.py", line 416, in scan_module
    code = marshal.load(f)
ValueError: bad marshal data (unknown type code)
```

**Comment by rogiervandergeer** [comment]
It seems the 3.7-dev is a fairly old development version. We could temporarily use the hack of 2139dc1, right?

**Comment by rogiervandergeer** [comment]
I like it

**Comment by koaning** [comment]
image

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 language: python
-python:
-  - "3.6"
+matrix:
+  include:
+    - python: 3.6
+    - python: 3.7
```
Comment (koaning) [inline .travis.yml]: yes

**Pull request merged**